en/orm/query-builder.rst

```diff
@@ -1119,6 +1119,8 @@ queries::
 
 .. _query-builder-updating-data:
 
+Note: Inserting records with the query builder will not trigger the ``Model.afterSave`` event. http://book.cakephp.org/3.0/en/orm/saving-data.html#inserting-data should be used instead if this is required.
```

Comment:
please don't add lines here (between the label and the title) as it will break an existing link in the  doc. Also please the the syntax reference to add a note and a link. 
http://book.cakephp.org/3.0/en/contributing/documentation.html

finally lines should be wrap to 80 chars.